core/src/main/java/org/apache/accumulo/core/clientImpl/ThriftTransportKey.java

```diff
@@ -91,7 +91,7 @@ public boolean equals(Object o) {
   }
 
   public final void precomputeHashCode() {
-    hashCode();
+    var unusedRetVal = hashCode();
```

Comment:
seems like in this case you could compute the `hash` in the ctor and get rid of this method

Reply:
Removed precomputeHashCode method and moved call to hashCode into constructors.